src/mongo/ICompositeRegen.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tripod\Mongo\Composites;
+
+/**
+ * Interface ICompositeRegen
+ * @package Tripod\Mongo\Composites
+ */
+interface ICompositeRegen
```

Comment:
Ultimately, I think it will (by the end of this PR).  Separate lets me do it only for Views until we're happy with the approach, then migrated it into `IComposite`.